hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMBlockProtocolServer.java

```diff
@@ -218,7 +218,7 @@ public void join() throws InterruptedException {
   @Override
   public List<DeleteBlockGroupResult> deleteKeyBlocks(
       List<BlockGroup> keyBlocksInfoList) throws IOException {
-    LOG.info("SCM is informed by OM to delete {} blocks", keyBlocksInfoList
+    LOG.debug("SCM is informed by OM to delete {} blocks", keyBlocksInfoList
```

Comment:
If changing to debug level, then better to add `if (LOG.isDebugEnabled())`?

Reply:
Thanks for @amaliujia's  prompt, judgment has been added.